liczek/src/main/java/pl/fratik/liczek/listeners/LiczekListener.java

```diff
@@ -19,6 +19,7 @@
 
 import com.google.common.eventbus.Subscribe;
 import net.dv8tion.jda.api.entities.ChannelType;
+import net.dv8tion.jda.api.entities.IFakeable;
```

Comment:
a po co to

Reply:
sprawdzałem jak mogę zobaczyć czy author jest webhookiem i najwidoczniej nie wyczyściłem importów